docs/static/pipeline-pipeline-config.asciidoc

```diff
@@ -169,14 +173,13 @@ An example of this pattern is in the following config:
 ----
 
 [[collector-pattern]]
-==== The collector pattern
-
-The collector pattern is used in situations where you want to define a common set of outputs and pre-output filters that many disparate pipelines might use. This is the opposite of the distributor pattern. In this pattern many pipelines fan in to a single pipeline where outputs and other processing are shared. This pattern simplifies configuration at the cost of reducing isolation, since all data is sent through a single pipeline.
+===== The collector pattern
 
-An example of this pattern can be seen below:
+Use the collector pattern when you want to define a common set of outputs and pre-output filters that many disparate pipelines might use. This pattern is the opposite of the distributor pattern. In this pattern many pipelines flow in to a single pipeline where they share outputs and processing. This pattern simplifies configuration at the cost of reducing isolation, since all data is sent through a single pipeline.
```

Comment:
I like how you make this description more active by saying "Use the collector pattern...."

I'd do something similar where you describe the other patterns.